src/dataviews/histogram-dataview-model.js

```diff
@@ -71,6 +71,14 @@ module.exports = DataviewModelBase.extend({
     if (attrs && (attrs.min || attrs.max)) {
       this.filter && this.filter.setRange(this.get('min'), this.get('max'));
     }
+
+    if (this.get('customStart')) {
```

Comment:
I need a custom property, or a flag to skip [this](https://github.com/CartoDB/carto.js/blob/master/src/dataviews/histogram-dataview-model.js#L338), otherwise when fetching the totals the `start` and `end` values will be overridden. I tried using a boolean, but it was harder to work with it that work with `customStart` and `customEnd`.